PROTO_tests/tests/logger_test.py

```diff
@@ -0,0 +1,100 @@
+
+    def test_arkouda_logger(self):
+        handler = StreamHandler()
+        handler.name = "streaming"
+        logger = getArkoudaLogger(name=self.__class__.__name__, handlers=[handler])
+        assert DEBUG == logger.level
+        assert "TestLogger" == logger.name
+        assert logger.getHandler("streaming") is not None
+        logger.debug("debug message")
+
+    def test_arkouda_client_logger(self):
+        logger = getArkoudaClientLogger(name="ClientLogger")
+        assert DEBUG == logger.level
+        assert "ClientLogger" == logger.name
+        logger.debug("debug message")
+        assert logger.getHandler("console-handler") is not None
+        with pytest.raises(ValueError):
+            logger.getHandler("console-handlers")
+
+    def test_update_arkouda_logger_log_level(self):
+        logger = getArkoudaLogger(name="UpdateLogger")
+        assert DEBUG == logger.level
+        logger.debug("debug before level change")
+        logger.changeLogLevel(LogLevel.WARN)
+        assert WARN == logger.handlers[0].level
+        logger.debug("debug after level change")
+
+        with tempfile.TemporaryDirectory(dir=TestLogger.logger_test_base_tmp) as tmp_dirname:
+            file_name = f"{tmp_dirname}/logger_test_output.txt"
+
+            handler_one = StreamHandler()
+            handler_one.name = "handler-one"
+            handler_one.setLevel(DEBUG)
+            handler_two = FileHandler(filename=file_name)
+            handler_two.name = "handler-two"
+            handler_two.setLevel(INFO)
+            logger = getArkoudaLogger(name="UpdateLogger", handlers=[handler_one, handler_two])
+            logger.changeLogLevel(level=LogLevel.WARN, handlerNames=["handler-one"])
+            assert WARN == handler_one.level
+            assert INFO == handler_two.level
+
+    def test_verbosity_controls(self):
+        logger = getArkoudaLogger(name="VerboseLogger", logLevel=LogLevel("INFO"))
+
+        assert INFO == logger.getHandler("console-handler").level
+        logger.debug("non-working debug message")
+        logger.enableVerbose()
+        assert DEBUG == logger.getHandler("console-handler").level
+        logger.debug("working debug message")
+        logger.disableVerbose()
+        assert INFO == logger.getHandler("console-handler").level
+        logger.debug("next non-working debug message")
+
+    def test_enable_disable_verbose(self):
+        loggerOne = getArkoudaLogger(name="loggerOne", logLevel=LogLevel.INFO)
+        loggerTwo = getArkoudaLogger("loggerTwo", logLevel=LogLevel.INFO)
+
+        loggerOne.debug("loggerOne before enableVerbose")
+        loggerTwo.debug("loggerTwo before enableVerbose")
+        ak.enableVerbose()
+        loggerOne.debug("loggerOne after enableVerbose")
+        loggerTwo.debug("loggerTwo after enableVerbose")
+        ak.disableVerbose()
+        loggerOne.debug("loggerOne after disableVerbose")
+        loggerTwo.debug("loggerTwo after disableVerbose")
```

Comment:
you missed converting these var names to use `_`

Reply:
Updated